src/quic.rs

```diff
@@ -9,17 +9,16 @@ use crate::key_schedule;
 use crate::session::{SessionCommon, Protocol};
 
 use std::sync::Arc;
-use ring::hkdf::{self, KeyType as _};
+use ring::hkdf;
 use webpki;
-use crate::msgs::base::PayloadU8;
 
 /// Secrets used to encrypt/decrypt traffic
-#[derive(Debug, Clone, Eq, PartialEq)]
+#[derive(Clone, Debug)]
```

Comment:
`Prk` doesn't implement `PartialEq` or `Eq` as of now.

Reply:
The way the code is structured requires `Prk` to implement `Clone` so I changed *ring* to do that. I hope we can eventually restructure things to avoid this cloning.